packages/liveblocks-react/src/comments/CommentsRoom.tsx

```diff
@@ -23,9 +23,9 @@ import React, {
   useEffect,
   useMemo,
 } from "react";
-import { useSyncExternalStore } from "use-sync-external-store/shim";
 import { useSyncExternalStoreWithSelector } from "use-sync-external-store/shim/with-selector.js";
 
+import { useSyncExternalStore } from "../factory";
```

Comment:
I've opted to instead import this from `factory.tsx` which is used in some of our other hooks and doesn't create build issues.